doc-Provisioning_Virtual_Machines_and_Hosts/topics/appe-cloudformation-template-service-provisioning.adoc

```diff
@@ -0,0 +1,90 @@
+[appendix]
+
+[[example-cloudformation-provisioning-via-services]]
+== Example: Using Amazon CloudFormation Template for Service Provisioning
+
+{product-title_short} supports Amazon CloudFormation orchestration templates that make deploying complex services easier in the cloud.
+
+[[example-adding-orchestration-template]]
+==== Creating an Orchestration Template
+
+Complete the following procedure to add new orchestration templates. 
+
+. Navigate to menu:Services[Catalogs] and select *Orchestration Templates* in the accordion menu.
+. Click image:1847.png[Configuration] *Configuration*, then click image:1848.png[Green_Plus_Sign] *Create a new Orchestration Template*. 
+. Enter a *Name* and *Description* for your template. 
+. Select *Amazon CloudFormation* from the *Template Type* list.
+. Select *Draft* to create a draft template.
+. Add your template in the area below for the selected *Template Type*. You can author your own stack template, or can add it here from other sources.
```

Comment:
Is there any way to add a template other than copy & pasting into the template field at the bottom of the editor? I think it might be useful to prompt users to copy & paste from an existing text file if its the easiest way to go about it. Honestly. the workflow at this step isn't entirely intuitive. I didn't even notice there was a field to input text at the bottom of the screen, probably because it doesn't have a header calling out the elements where users add or draft a template. I'll raise this as an issue.

Reply:
Agree; fixed. Good call on the UI issue.